src/components/notification-banner/index.md.njk

```diff
@@ -41,6 +41,12 @@ Avoid showing more than one notification banner on the same page. Instead, combi
 
 Do not show a notification banner and an [error summary](/components/error-summary/) on the same page. Show the error summary instead of the notification banner.
 
+### Notification banner headings
+
+You can add `<h3>` headings to the `govuk-notification-banner__content` container if you have structured content. 
+
+For single-line notifications that have no structure avoid using headings.
```

Comment:
```suggestion
You can use `<h3>` headings to the `govuk-notification-banner__content` to help structure your content. 

Avoid using headings for single-line notifications that have no structure.
```

Reply:
Suggested update to clarify what "structure" means in the context of banner content

Reply:
Yeah I wonder if 'have no structure' is accurate really, what does that mean?

Maybe something like 'Avoid using headings for single-line notifications that don't need one'?

Reply:
Perfect!

Reply:
Agreed! Updated suggestion above ⬆️ , feel free to commit the suggestion and merge.